uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+use Civi\Test;
+use CRM_HRCore_Service_DrupalUserService as DrupalUserService;
+use CRM_HRCore_Service_DrupalRoleService as DrupalRoleService;
+
+/**
+ * @group headless
+ */
+class DrupalUserServiceTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
```

Comment:
Would be nice to have considering the fact that we will have to keep replicating all these information in all the test classes we add just like you have in the test class for Drupal Role service. Also all the things we need to install will be in one place rather than adding it per test class.